native/vendor/google/play/PlayTask.h

```diff
@@ -0,0 +1,89 @@
+/****************************************************************************
+ Copyright (c) 2025 Xiamen Yaji Software Co., Ltd.
+
+ http://www.cocos.com
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights to
+ use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ of the Software, and to permit persons to whom the Software is furnished to do so,
+ subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+****************************************************************************/
+
+#pragma once
+
+#include <unordered_map>
+#include "base/RefCounted.h"
+#include "vendor/google/common/ScopedListener.h"
+
+namespace se {
+class Object;
+}
+namespace cc {
+class TaskException : public RefCounted {
+public:
+    ~TaskException();
+    std::string getMessage() const {

```

Comment:
Return `const std::string &` to avoid string copy.

Reply:
Shouldn't it be avoided here? Because in the auto-binding code, it is the string that is defined to assign the value?
![image](https://github.com/user-attachments/assets/5520a02c-1421-4bd3-a2c5-a35a09ec2287)

Reply:
Assign `const std::string &` to `std::string` will do a copy.
At the point of view `TaskException::getMessage` API, it's better to return `const &` since you don't know how user uses this API. It's could be 

```c++
const auto &result = xxx->getMessage();
```

or the binding usage:

```
std::string result = xxx->getMessage();
```

That depends on the usage case.

Reply:
Yes, it's been fixed.